ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -1633,36 +1633,60 @@ public class GlobalTokens: NSObject {
     // MARK: - IconSize
 
     public enum IconSizeToken: TokenSetKey {
-        case xxxSmall
-        case xxSmall
-        case xSmall
-        case small
-        case medium
-        case large
-        case xLarge
-        case xxLarge
-        case xxxLarge
+        case sizeNone
+        case size20
+        case size40
+        case size60
+        case size80
+        case size100
+        case size120
+        case size160
+        case size200
+        case size240
+        case size280
+        case size320
+        case size360
+        case size400
+        case size480
+        case size520
+        case size560
```

Comment:
So my understanding of the new size ramp that was recently created is that it was primarily meant to replace the existing spacing ramp. On FURN, we're currently replacing all the existing references to globalTokens.spacing with globalTokens.size. There are currently no global tokens specific to icon size. Is the new size ramp also meant to be used for icon size, or should there be an entirely different ramp for icon size?

Reply:
In FUA we're using both spacing and iconSize. Spacing is mostly used for paddings though. I agree it'd be a good idea to just use size since all but one of the spacing tokens are included in the new size tokens, and that will probably save us some binary as well. Out of curiosity, did design explicitly they wanted us to use size tokens for everything?

Reply:
Icon size tokens haven't been talked about at all in the FURN token discussions - if there haven't been conversations with design about this on the FUA side I think this would be good to bring up in the Fluent Apple x Design Discussion channel - i.e. should the existing size ramp also be used for icon size? Or are there plans for a new ramp specific to icon size